app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ProjectsController < ApplicationController
+  @@MAX_VOTES = 3
+
+  def create
+    @hackday = Hackday.find(params[:hackday_id])
+    @project = @hackday.projects.build(filter_params)
+    if @project.save
+      redirect_to hackday_path(@hackday)
+    else
+      @hackday.projects.delete(@project)  # unassociate so it doesn't show in the list?
```

Comment:
you should just do @hackday.reload here or something like that. delete is confusing and not exactly what you want.